src/scss/components/_methodology.scss

```diff
@@ -0,0 +1,57 @@
+@use 'abstracts' as *;
+
+.methodology {
+  position: relative;
+  display: grid;
+  grid-column-gap: 72px;
+  grid-template-columns: 2fr 1fr;
+  padding-top: rem(60px);
+
+  &::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: calc((100vw - 100%) / -2 - 10px);
+    display: block;
+    width: 100vw;
+    height: 1px;
+    background-color: #979797;
```

Comment:
Can you check if this is the same color as the one used for the header? If it is, we should make this a color variable.